maven/impl-maven-embedded/src/main/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/BuildTrigger.java

```diff
@@ -39,10 +39,10 @@ private void setOutputHandlers(String expectedRegex, CountDownLatch countDownLat
                 new ResolverErrorOutputHandler(logBuffer, expectedRegex, countDownLatch);
             ResolverOutputHandler outputHandler = new ResolverOutputHandler(logBuffer, expectedRegex, countDownLatch);
 
-            invoker.setOutputHandler(outputHandler);
+            invocationRequest.setOutputHandler(outputHandler);
```

Comment:
This seems to be duplicated

Reply:
Hmmm... I don't see where / how

Reply:
L42 and L43 are the same - `invocationRequest.setOutputHandler(outputHandler);`
L45 and L46 are the same - `invocationRequest.setErrorHandler(errorOutputHandler);`

Or is it necessary to set the handlers 2 times?

Reply:
Oh, I see it now. Thanks for pointing this out.
Fixed!